ovs/lib/helpers/generic/scrubber.py

```diff
@@ -620,13 +620,17 @@ def _verify_mds_config(current_vdisk):
                             # Do the actual scrubbing
                             with vdisk.storagedriver_client.make_locked_client(str(vdisk.volume_id), update_interval_secs=lease_interval) as locked_client:
                                 self._logger.info('{0} - Retrieve and apply scrub work'.format(vdisk_log, vdisk.name))
+                                starttime = time.time()
+                                graphite_controller = GraphiteController()
```

Comment:
A single controller for the whole scrub job should be enough.
Declare it in the class Scrubber and pass it along to all threads